differentiable_robot_model/data_utils.py

```diff
@@ -10,6 +10,9 @@
 from torch.utils.data.dataset import Dataset
 
 
+torch.set_default_tensor_type(torch.FloatTensor)
```

Comment:
Are we allowing users to set default tensor type for GPU-based experiments? If so we shouldn't be setting the default tensor type here.

Reply:
oh I think this was a left over line, the code wasn't working on Mac anymore after the GPU PR, but then your new test PR fixed things (which I merged into my branch), and I forgot to remove my "fixes"